datasets/ru/cbr_banks/ru_cbr_banks.yaml

```
@@ -0,0 +1,45 @@
+title: Russia Banking Registry
+entry_point: crawler.py
+prefix: ru-bic
```

Let's call this bik rather than bic to avoid confusion with SWIFT, even though their API calls refer to bic

```suggestion
prefix: ru-bik
```